### PR TITLE
phpunit 9.3.2

### PR DESCRIPTION
**Pull request by bepsvpt**
---

Debug Info:
- homebrew updater version: 1.0.6
- formula new file size: 4,133,631 bytes
- formula fetch time: 4.7 seconds

Pull request opened by [homebrew-updater](https://github.com/bepsvpt/homebrew-updater) project.

Open a new [issue](https://github.com/bepsvpt/homebrew-updater/issues) to monitor new formula.

### CODE REVIEW

**Comment by chenrui333** [comment]
@bepsvpt, 👋 , thanks for the PR, can you rebase the branch to the latest master? I was trying to help out, but had some permission denial issue.

**Comment by bepsvpt** [comment]
@chenrui333 

Done!

**Comment by chenrui333** [comment]
> @chenrui333
> 
> Done!

please do `rebase` instead of `merge master`, `merge master` does not fit into our workflow. Thanks!

File: Formula/phpunit.rb [hunk 1]
```diff
@@ -1,9 +1,8 @@
 class Phpunit < Formula
   desc "Programmer-oriented testing framework for PHP"
   homepage "https://phpunit.de"
-  url "https://phar.phpunit.de/phpunit-9.2.6.phar"
-  sha256 "8223865d3de82828009fb07abd3ab7640f6c58310c62a46a2ce98bce5655f295"
-  license "BSD-3-Clause"
```
Comment (chenrui333) [inline Formula/phpunit.rb:6]: looks like the license got removed 😅

**Pull request closed without merging**